packages/jetpack-core/composer.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "automattic/jetpack-core",
+	"description": "Core library for Jetpack",
+	"homepage": "http://jetpack.com/",
+	"version": "7.2-alpha",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"support": {
+		"issues": "https://github.com/Automattic/jetpack/issues"
+	},
+	"require-dev": {
+		"dealerdirect/phpcodesniffer-composer-installer": "0.5.0",
+		"wp-coding-standards/wpcs": "2.0.0",
+		"sirbrillig/phpcs-variable-analysis": "2.6.0",
+		"phpcompatibility/phpcompatibility-wp": "2.0.0"
+	},
+	"scripts": {
+		"php:compatibility": "composer install && vendor/bin/phpcs -p --runtime-set testVersion '5.2-' --standard=PHPCompatibilityWP --ignore=docker,tools,tests,node_modules,vendor --extensions=php",
```

Comment:
This is obviously work in progress, but the testVersion could be updated to 5.3 given that the work here would rely on namespaces... or 5.6 which would become the minimun on WordPress soon.

Reply:
+1 to 5.6. Anything we do in this realm wouldn't be in a ready to ship state until we're on at least WP 5.2 as a minimum thus supporting modern PHP only.